bscript/interpreter/doc.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2013-2017 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+/*
+Package interpreter implements the bitcoin transaction script language.
+
+A complete description of the script language used by bitcoin cash can be found at
```

Comment:
we should prolly also remove traces of "bitcoin cash" haha had enough of that baggage 😅